<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-5GBS83J" ignored="3" name="Test" tests="1" failures="1" timestamp="2021-01-28T10:56:55 IST" time="120.556" errors="0">
  <testcase name="testcase4" time="120.387" classname="framework.Appium.Demo">
    <failure type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException" message="The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\HariniKantheti\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1] 
">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\HariniKantheti\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1] 

at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:168)
at framework.Appium.Capability.startServer(Capability.java:41)
at framework.Appium.Demo.testcase4(Demo.java:100)
at java.util.ArrayList.forEach(Unknown Source)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/wd/hub/status] to be available after 120027 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:133)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:155)
... 30 more
Caused by: java.util.concurrent.TimeoutException
at java.util.concurrent.FutureTask.get(Unknown Source)
at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:156)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
... 32 more
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testcase4 -->
  <testcase name="testcase3" time="0.0" classname="framework.Appium.Demo">
    <ignored/>
  </testcase> <!-- testcase3 -->
  <testcase name="testcase2" time="0.0" classname="framework.Appium.Demo">
    <ignored/>
  </testcase> <!-- testcase2 -->
  <testcase name="testcase1" time="0.0" classname="framework.Appium.Demo">
    <ignored/>
  </testcase> <!-- testcase1 -->
</testsuite> <!-- Test -->
